package_test.go

```diff
@@ -52,6 +52,9 @@ func TestPackageBinfile(t *testing.T) {
 		}
 		got := pkg.Binfile()
 		want := filepath.Join(ctx.Bindir(), tt.want)
+		if pkg.gotargetos == "windows" {
```

Comment:
I wasn't entirely sure what `tt.goos` is for, but it's not set to `"windows"` so I went for `pkg.gotargetos` which was appropriately set to `"windows"`. :confused: